packages/server/beaver_triple_service/cmd/generate_jwt.go

```diff
@@ -0,0 +1,175 @@
+package cmd
+
+import (
+	"bufio"
+	"bytes"
+	"crypto/rand"
+	"encoding/base64"
+	"encoding/json"
+	"fmt"
+	"log"
+	"os"
+	"path"
+	"strings"
+
+	"github.com/golang-jwt/jwt/v4"
+	"github.com/google/uuid"
+	"github.com/spf13/cobra"
+	"gopkg.in/yaml.v3"
+
+	jwt_types "github.com/acompany-develop/QuickMPC/packages/server/beaver_triple_service/jwt"
+)
+
+type Options struct {
+    file_path  string
+    output_dir string
+}
+
+var (
+    o = &Options{}
+)
+
+func store_client_env(path string, token string) error {
```

Comment:
```suggestion
func storeClientEnv(path string, token string) error {
```

Reply:
it is written by me :bow:

Reply:
Corrected with [2db2237](https://github.com/acompany-develop/QuickMPC/pull/123/commits/2db2237f5ea6086d36d45d46db563493754334a3)